catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -48,9 +51,12 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(QueryApplication.class);
 
   private static final String APPLICATION_JSON = "application/json";
+  private static final String TRANSFORMER_ID_PROPERTY = "id";
```

Comment:
✏️ We've already defined a constant for this. https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-api/src/main/java/ddf/catalog/Constants.java#L43